service/src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -474,8 +477,10 @@ public boolean updateResource(
       UUID resourceId,
       @Nullable String name,
       @Nullable String description,
-      @Nullable String attributes) {
-    return updateResourceWorker(workspaceId, resourceId, name, description, attributes);
+      @Nullable String attributes,
+      @Nullable CloningInstructions cloningInstructions) {
```

Comment:
It's a little concerning to me that changing this signature doesn't require updating any unit tests. Out of scope for this PR, but I think our unit test coverage needs some work